src/backend/gl/src/lib.rs

```diff
@@ -745,3 +745,7 @@ impl gfx_core::DeviceFence<Resources> for Device {
         }
     }
 }
+
+mod gl {
+    include!(concat!(env!("OUT_DIR"), "/gl_bindings.rs"));
```

Comment:
Sry, haven't realized that. Though it appears only the performance example uses it. I'll revert it back to using gfx-gl for now.